server/views/pages/overview/partials/overviewTabContents.njk

```diff
@@ -2,4 +2,40 @@
   <div class="govuk-grid-column-two-thirds">
     {% include './overviewTabActionPlanColumn.njk' %}
   </div>
+
+  <div class="govuk-grid-column-one-third">
+
+    <div class="govuk-summary-card">
+      <div class="govuk-summary-card__title-wrapper app-summary-card__title-wrapper--white">
+        <h2 class="govuk-summary-card__title">Current functional skills</h2>
+      </div>
+      <div class="govuk-summary-card__content">
+
+        <table class="govuk-table govuk-!-margin-bottom-0">
+          <caption class="govuk-table__caption govuk-visually-hidden">Functional skills and assessment information</caption>
+          <thead class="govuk-table__head govuk-visually-hidden">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header">Skill and assessment date</th>
+              <th scope="col" class="govuk-table__header">Assessment level</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            {% for skill in functionalSkills.assessments %}
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header">
+                <span class="govuk-body-s">Assessed on {{ skill.assessmentDate | formatDate('D MMMM YYYY') }}</span>
+                <br>
+                {{ skill.type | replace("_", " ") | title }}
```

Comment:
We already have a filter for rendering the skill type (its not quite as simple as replacing the spaces with underscores); plus ... I think we're going to need to understand the design in respect of what happens when a functional skill has no recorded assessment date or grade .... 🤔  .... basically the "business rules" 🙄 say that English and Maths are always reported, even if there are no assessments for them yet!

See this (which also includes the filter) in `educationAndTrainingTabContents.njk` :
```
        <tbody class="govuk-table__body">
        {% for assessment in functionalSkills.assessments %}
          <tr class="govuk-table__row">
            <td class="govuk-table__cell">{{ assessment.type | formatFunctionalSkillType }}</td>
            {% if assessment.assessmentDate and assessment.grade %}
              <td class="govuk-table__cell">{{ assessment.assessmentDate | formatDate('D MMMM YYYY') }}</td>
              <td class="govuk-table__cell">{{ assessment.grade }}</td>
              <td class="govuk-table__cell">Induction</td>
            {% else %}
              <td class="govuk-table__cell" colspan="3">No functional skills assessment recorded</td>
            {% endif %}
          </tr>
        {% endfor %}
        </tbody>
```

If you use Zebedee Zebra you'll see what I mean about always reporting English and Maths even if there are no assessments recorded yet